src/app/[locale]/specs/xrpc/page.mdx

```diff
@@ -87,13 +89,17 @@ The current mechanism is to use short-lived JWTs signed by the account's atproto
 
 The JWT parameters are:
 
-- `alg` header field: indicates the signing key type (see [Cryptography](/specs/cryptography))
+- `alg` header field (string, required): indicates the signing key type (see [Cryptography](/specs/cryptography))
     - use `ES256K` for `k256` keys
     - use `ES256` for `p256` keys
-- `iss` body field: account DID that the request is being sent on behalf of. This may include a suffix service identifier; see below
-- `aud` body field: service DID associated with the service that the request is being sent to
-- `exp` body field: token expiration time, as a UNIX timestamp with seconds precision. Should be a short time window, as revocation is not implemented. 60 seconds is a good value.
-- JWT signature: base64url-encoded signature using the account DID's signing key
+- `typ` header field (string, required): `JWT`
```

Comment:
I don't think we have formally intended to specify that this must be "JWT".  We intentionally do not require that it take this value when we verify the token since we hadn't explicitly specified it.  Wanted to leave the door open to a different value we might prefer more, e.g. perhaps something akin to `at+jwt`/`refresh+jwt`/`dpop+jwt`.

Reply:
@matthieusieben had recommended this in https://github.com/bluesky-social/atproto-website/issues/327.

I can mention the field and that it doesn't have a stable/specified value yet for this use-case?

Reply:
I indeed wanted to put something there. Whether it's JWT or something more specific is a more complex topic.

Either we consider that there will only ever be only one type of JWT in Atproto (besides access & refresh tokens) and `JWT` is fine. Or we want to be future proof and we might want to do something like `service+jwt` (or `pds-2-pds+jwt`, or whatever).

The important thing here is to make sure that a JWT generated for one particular purpose cannot be used for another purpose, while still allowing them to be signed with the same key
 a dev could choose to implement a PDS from scratch using the same key everywhere
 The only control we'd have is the typ header.

Reply:
I updated with:

> currently `JWT`, but intend to update to a more specific value